src/adagrams.js

```diff
@@ -1,7 +1,130 @@
 const Adagrams = {
+  lettersFrequency: { A: 9, B: 2, C: 2, D: 4, E: 12, F: 2, G: 3, H: 2, I: 9, J: 1,
+    K: 1, L: 4, M: 2, N: 6, O: 8, P: 2, Q: 1, R: 6, S: 4, T: 6,
+    U: 4, V: 2, W: 2, X: 1, Y: 2, Z: 1 },
+
   drawLetters() {
     // Implement this method for wave 1
+    let letterArray = [];
+    for (const letter in this.lettersFrequency) {
+      for (let i = 1; i <= this.lettersFrequency[letter]; i++){
+        letterArray.push(letter);
+      }
+    }
+    let lettersInHand = [];
+    for (let i = 0; i < 10; i++) {
+      let randomLetter = letterArray[Math.floor(Math.random() * letterArray.length)];
+      lettersInHand.push(randomLetter);
+    }
+    return lettersInHand;
+  },
+
+  usesAvailableLetters(input,lettersInHand) {
```

Comment:
👍 Just be aware that this changes the `lettersInHand` array, so the function that calls `usesAvailableLetters` has their argument modified.